### DISCUSSION TITLE
Keep 0.102 Heatmap as a backlight option for 0.107 Heatmap not compatible models

### DISCUSSION THREAD

**Discussion by Kowken** [seerge/g-helper#1027] [Ideas]
Greetings.

I already posted on this sub regarding issue #967 and while the heatmap mode works on my keyboard, it also activates my keystone red LED light and does not set it off when I change settings.

Thus I would like to know if it was possible to keep this option alongside the new one as 0.102 heatmap blinking is barely noticeable (or at least a guide to build new releases with the old heatmap).

Thanks in advance for your answer and have a nice day.
Best regards.

(I posted this in ideas and not issues because #967 failed to address this peculiar problem and that the previous solution was satisfactory)

**Comment by seerge** [comment]
Specially for you

Add ``"old_heatmap" : 1,`` to ``%AppData%\GHelper\config.json``

And use this build, it will keep using old heatmap (i will keep this option in next releases as well)

[GHelper.zip](https://github.com/seerge/g-helper/files/12290871/GHelper.zip)

**Reply by Kowken** [reply]
Thank you very much ! Does the modification in the config.json file work for release 0.107 or will it work onwards ?
Also is the fact that the GHelper.dll.config file is absent from this build normal ? (Just wondering)

**Reply by seerge** [reply]
It won’t work in 107, but will in future releases. Dll.config is irrelevant

**Reply by Kowken** [reply]
Noted. Also I will let you know whenever I manage to shut down the Keystone LED. Have a nice day :)
